collector/logical_router_port_collector.go

```diff
@@ -26,8 +26,18 @@ type logicalRouterPortCollector struct {
 	txTotalByte     *prometheus.Desc
 }
 
-func newLogicalRouterPortCollector(apiClient *nsxt.APIClient, logger log.Logger) prometheus.Collector {
+type logicalRouterPortStatisticMetric struct {
+	LogicalRouterPort manager.LogicalRouterPort
+	Rx                *manager.LogicalRouterPortCounters
```

Comment:
nit: should do the golang (or networking?) way
```suggestion
	RX                *manager.LogicalRouterPortCounters
```

Reply:
I was thinking RX is not an initial, it should be considered as a single word
Is there any example on the official Go that uses Rx/RX?
I've found some conflicting type in the community repo

Reply:
found some with RX/TX
https://github.com/akhenakh/statgo/blob/3ae2cda264c55e1eaffaaa11f626b6c833eeff29/net_io_stats.go#L13

another one with Rx/Tx
https://github.com/bcicen/ctop/blob/a2c08d312ef9db1e7dad162c5c47981abab940a6/models/main.go#L10

I guess it doesn't really matter then, let's just stick to what we have